test/integrationTests/launchConfiguration.integration.test.ts

```diff
@@ -0,0 +1,48 @@
+/*--------------------------------------------------------------------------------------------- 
+ *  Copyright (c) Microsoft Corporation. All rights reserved. 
+ *  Licensed under the MIT License. See License.txt in the project root for license information. 
+ *--------------------------------------------------------------------------------------------*/
+
+import * as fs from 'async-file';
+import * as vscode from 'vscode';
+
+import poll from './poll';
+import { should } from 'chai';
+import testAssetWorkspace from './testAssets/testAssetWorkspace';
+
+const chai = require('chai'); 
+chai.use(require('chai-arrays')); 
+chai.use(require('chai-fs')); 
+ 
+suite(`Tasks generation: ${testAssetWorkspace.description}`, function() {
+    suiteSetup(async function() { 
+        should();
```

Comment:
Did you want to call `should()` by itself?

Reply:
This is the same pattern as `common.test.ts`, `json.test.ts`, etc. I'm just happening to use a function instead of a lambda here, right?

Reply:
Oh, got it.